samples/snippets/src/main/java/pubsublite/kafka/KafkaIOReadExample.java

```diff
@@ -0,0 +1,118 @@
+package com.google.cloud.pubsublite.demo;
```

Comment:
Shouldn't use `com.google.*` for sample / demo package names.  `com.example.*` or `productname.xxx`

Reply:
We don't want customers accidently publishing w/ com.google